Dockerfile.gpu

```diff
@@ -44,7 +44,11 @@ ARG TENSORFLOW_PIP
 ARG TORCH_PIP
 ARG TORCHVISION_PIP
 ARG LIGHTNING_PIP
+ARG TF_CUDA_SYM
+
 RUN if [ "$TENSORFLOW_PIP" ]; then python -m pip install $TENSORFLOW_PIP; fi
+RUN if [ "$TF_CUDA_SYM" ]; then /tmp/det_dockerfile_scripts/tf2.4-cuda11.1-symlink.sh; fi
```

Comment:
explanation for this: cuda 11.0 uses a math package cusolver 10.0, cuda 11.1 comes with cusolver 11. tensorflow 2.4 is looking for cusolver 10.0. putting a soft link between the LD_LIBRARY_PATH cusolver and tensorflow's cusolver dependency fixes the issue. cusolver 11 seems to be backwards compatible with cusolver 10. there doesn't appear to be any issues so far.